src/Tribe/Views/V2/Views/Widgets/Widget_View.php

```diff
@@ -61,6 +62,37 @@ protected function setup_template_vars() {
 		return $template_vars;
 	}
 
+	/**
+	 * Sets up the View repository arguments from the View context or a provided Context object.
+	 *
+	 * @since 4.9.3
+	 *
+	 * @param  Context|null $context A context to use to setup the args, or `null` to use the View Context.
+	 *
+	 * @return array The arguments, ready to be set on the View repository instance.
+	 */
+	protected function setup_repository_args( Context $context = null ) {
+		bdump('widget_setup_repository_args');
+		$context = null !== $context ? $context : $this->context;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: 0e36dc6e1ee79763ec86060a8095ede8</sub>